packages/twenty-server/src/utils/parse-email-body.ts

```diff
@@ -0,0 +1,11 @@
+import { type JSONContent } from 'twenty-emails';
+
+export const parseEmailBody = (body: string): JSONContent | string => {
```

Comment:
Seems like a too high level util for what you're making of it. You should either rename to something more generic so it can be reused like `parseJsonOrFallbackToString` or otherwise you should make it within a subfolder of emails and keep it named this way if the logic will become specific to that module